src/app/client/projects/desktop/src/app/modules/content-consumption/components/credits-and-licence/credits-and-licence.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input } from '@angular/core';
+import { ResourceService } from '@sunbird/shared';
+
+@Component({
+  selector: 'app-credits-and-licence',
+  templateUrl: './credits-and-licence.component.html',
+  styleUrls: ['./credits-and-licence.component.scss']
+})
+export class CreditsAndLicenceComponent {
+
+  @Input() contentData;
+  constructor(public resourceService: ResourceService) { }
+
+  embedUrl(text: string) {
+    if (text) {
+      const urlRegex = /(https?:\/\/[^\s]+)/g;
+      return text.replace(urlRegex, '<a href="$1">$1</a>');
```

Comment:
What happens when we click on the url ?

Reply:
I missed that scenario, but I think we should open it in the browser.